test/controllers/income/UpdateIncomeNextYearControllerSpec.scala

```diff
@@ -60,7 +60,7 @@ class UpdateIncomeNextYearControllerSpec
 
   "onPageLoad" must {
     "redirect to the duplicateSubmissionWarning url" when {
-      "an income update has already been performed" in {
+      "an income update has already been performed" ignore {
```

Comment:
Any reason we are ignoring this test?

Reply:
Yeah will be brought back in as part of https://jira.tools.tax.service.gov.uk/browse/DDCNL-1458

Reply:
Sound, no worries